### ISSUE TITLE
Druid Feat "Awaken The Beast" Increases HP Per Level Instead of Wildshape HP

### ISSUE THREAD

**Issue by tmachad** [SolastaMods/SolastaUnfinishedBusiness#2817]
The druid feat "Awaken The Beast" seems to be increasing the druid's base HP per level instead of their wildshape HP. Noticed this when I loaded a save and the level 6 druid suddenly had twice as much HP as he should have.

This commit seems like a likely culprit: https://github.com/SolastaMods/SolastaUnfinishedBusiness/commit/add09bf3f3edb29b22aedebefe5bf151150bcc3b

**Comment by ThyWoof** [comment]
Can I have a save? Saves (no pun intended) a lot of time on our end.

**Comment by tmachad** [comment]
Absolutely, here you go: [Awaken_The_Beast.zip](https://github.com/SolastaMods/SolastaUnfinishedBusiness/files/11050059/Awaken_The_Beast.zip)

**Comment by ThyWoof** [comment]
Thanks. That commit wasn’t the cause by the way. Gotta fully rework this feat.

**Comment by tmachad** [comment]
Oh well, I'm not familiar with solasta modding but I figured I'd take take a guess. Thanks for looking into the issue!

**Comment by ImAkemi** [comment]
Came here to report the same issue, luckily you can go into their stats and reduce their HP per level to counteract the issue. My druid was gaining 8hp per level for some reason.

**Comment by ThyWoof** [comment]
I have it fixed but still need advice on one final detail. Game by default move conditions back from beast to Druid. Should I cancel this temp hp form in that scenario? Also, for how long should these temp hp be granted?

**Comment by tmachad** [comment]
I think the intuitive behavior would be for the temp hp to last as long as the beast form, no more, no less. That being said, I'm not an expert in D&D or solasta modding, so I don't know how well that maps to the rules of D&D or the internal mechanics of solasta.

**Comment by ThyWoof** [comment]
This is a homebrew feat and I agree it should terminate . Will check how hard

**Comment by ThyWoof** [comment]
Fixed [here](https://github.com/SolastaMods/SolastaUnfinishedBusiness/commit/4b29cc258bfbe7864ce89398875c66c6e7ec062b). It'll remove temp HP on revert shape. Duration is half Druid levels hours.
